code/game/jobs/job/job.dm

```diff
@@ -203,24 +203,16 @@
 		entry_message_intro = "You are the [title]!"
 	if(!entry_message_end)
 		entry_message_end = "As the [title] you answer to [supervisors]. Special circumstances may change this!"
-	return "[entry_message_intro]<br>[entry_message_body]<br>[entry_message_end]"
+	return "[role_header(entry_message_intro)][role_body(entry_message_body)][role_body(entry_message_end)]"
 
 /datum/job/proc/announce_entry_message(mob/living/carbon/human/H, datum/money_account/M, whitelist_status) //The actual message that is displayed to the mob when they enter the game as a new player.
 	set waitfor = 0
 	sleep(10)
 	if(H && H.loc && H.client)
-		var/title_given
-		title_given = lowertext(disp_title)
 
 		//Document syntax cannot have tabs for proper formatting.
-		var/entrydisplay = " \
-			[SPAN_ROLE_BODY("|______________________|")] \n\
-			[SPAN_ROLE_HEADER("You are \a [title_given]")] \n\
-			[flags_startup_parameters & ROLE_ADMIN_NOTIFY ? SPAN_ROLE_HEADER("You are playing a job that is important for game progression. If you have to disconnect, please notify the admins via adminhelp.") : ""] \n\
-			[SPAN_ROLE_BODY("[generate_entry_message(H)]<br>[M ? "Your account number is: <b>[M.account_number]</b>. Your account pin is: <b>[M.remote_access_pin]</b>." : "You do not have a bank account."]")] \n\
-			[SPAN_ROLE_BODY("|______________________|")] \
-		"
-		to_chat_spaced(H, html = entrydisplay)
+		var/entrydisplay = "[generate_entry_message(H)][M ? role_body("Your account number is: <b>[M.account_number]</b>. Your account pin is: <b>[M.remote_access_pin]</b>.") : role_body("You do not have a bank account.")][flags_startup_parameters & ROLE_ADMIN_NOTIFY ? role_header("<big>You are playing a job that is important for game progression. If you have to disconnect, please notify the admins via adminhelp.</big>") : ""]"
```

Comment:
This really needs to be broken up into multiple variables that you can concatenate together in the following to_chat call. E.g. `var/account` and `var/important` and then the to_chat has `[generate_entry_message(H)][account][important]` If there's no account then leave account as `""`; same for important.

It used to be clear what was on each line, now its just a giant line of text spanning over 400 characters (used to be at worst ~200 characters) in code with multiple large ternary conditional operators.

Reply:
Yeah that whole message is fairly ugly as is. I'll take a look at it when I have a moment and see what I can come up with.